diablo/routes.py

```diff
@@ -72,6 +74,17 @@ def front_end_route(**kwargs):
         vue_base_url = app.config['VUE_LOCALHOST_BASE_URL']
         return redirect(vue_base_url + request.full_path) if vue_base_url else make_response(index_html)
 
+    @app.errorhandler(Exception)
+    def handle_exception(e):
+        subject = str(e)
+        if isinstance(e, HTTPException):
+            # No Ops notification when HTTP error
+            app.logger.exception(e)
+        else:
+            # Notify Diablo Ops teams
+            send_system_error_email(message=f'{subject}\n\n{traceback.format_exc()}', subject=f'{subject[:20]}...')
+        return {'message': subject}, 500
+
```

Comment:
This is fix for https://jira.ets.berkeley.edu/jira/browse/DIABLO-366  Test coverage in next PR.